openmetadata-service/src/main/java/org/openmetadata/service/migration/utils/v150/MigrationUtil.java

```diff
@@ -138,6 +138,13 @@ public static void deleteLegacyDataInsightPipelines(
     }
   }
 
+  public static void updateEmailTemplates(CollectionDAO collectionDAO) {
+    CollectionDAO.DocStoreDAO dao = collectionDAO.docStoreDAO();
+
+    // delete emailTemplates, it will be fetched from initSeedData
+    dao.deleteEmailTemplates();
```

Comment:
this assumes that people are stopping and doing migrations. Lets do the migration of deleting and updating the email template as part of this, instead of relying the server to load data

Reply:
Noted.